.github/workflows/build.yml

```diff
@@ -64,6 +64,38 @@ jobs:
           cd build
           ctest --output-on-failure
 
+  ubuntu-latest-cmake-sanitizer:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        sanitizer: [ADDRESS, THREAD, UNDEFINED]
```

Comment:
I don't have a source for this at hand, and i don't thin you should change it, but address and ub sanitizers can be combined. (just so you know for your dev setup :wink: )

Reply:
Good to know, never used those. Had a look and tbh they seem to have useful features